packages/Progress/Bar/textures/Disabled.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const Disabled = () => (
+  <svg width="100%" height="24">
+    <defs>
+      <pattern id="zohoz" patternUnits="userSpaceOnUse" width="8" height="8">
```

Comment:
It might, but as far as I'm aware it has no actual consequences. I'm not sure how we could refactor this besides generating a random ID for every rendered component, which adds extra maintenance load. The `path` needs to reference the `id` to properly use the `fill` css property.

Reply:
I think we should use `uniqueId` from `@tds/core-util-helpers`. We've run into this problem before where pipelines will fail due to a11y errors.